### ISSUE TITLE
SS01S Compatibility

### ISSUE THREAD

**Issue by iKrushYou** [paullj1/TreatLife-HomeKit#5]
Hey, I'd love to add in compatibility for the [SS01S Switch](https://www.amazon.com/Treatlife-Smart-Light-Switch-Assistant/dp/B07R4MFWCQ/ref=sr_1_11?keywords=treatlife&qid=1640187744&sr=8-11)

I'm more than happy to create the PR myself and I've started trying to work on the code but would you happen to have any pointers? I started off by flashing the DS02S firmware just to see but that doesn't work for obvious reasons. 

I know need to remove the "dimmer" functionality and possibly update the proper GPIO

I already had these switches configured with Tasmota like so
`{"NAME":"Treatlife Switch","GPIO":[0,0,0,0,288,576,1,1,224,160,0,0,0,1],"FLAG":0,"BASE":18}`

and hopefully that would help identify the proper GPIO 

Anyway, in short, looking for some pointers as to where to update the GPIO values (I haven't found it yet in the code) 

~oh, also, I'm assuming when developing you override the wifi so it doesn't go to the capture portal every time, I will look into how to do that but if you see this before I figure it out that would be helpful too~
I think I got it `wifiManager.autoConnect("AutoConnectAP", "password")`

**Comment by paullj1** [comment]
Hey Alex!  Since the SS01S just has a simple relay, I typically just use the [HomeKit Accessory Architect (HAA)](https://github.com/RavenSystem/esp-homekit-devices) firmware.  While we could add it here, the whole point of this project was to expose the Tuya serial protocol to HomeKit (which isn’t really a thing for those on/off relay switches… just a GPIO output).

Hope that makes sense, and helps!

**Comment by iKrushYou** [comment]
> 

Thanks! This is so much better!

I really appreciate the quick reply saving me a lot of time
